roles/selinuxoff/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: selinuxoff
+  command: echo SELINUX=disabled >> /etc/selinux/config
```

Comment:
This can be done with the selinux module in Ansible. See the top of https://github.com/platform9/express/blob/master/roles/neutron-prerequisites/tasks/redhat.yml